File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/Delphi/UnboundDesignDefinedDemo/UnboundDesignDefinedDemoMain.pas
unit UnboundDesignDefinedDemoMain;

interface

uses
  Windows, Forms, Messages, SysUtils, Classes, ActnList, ImgList, Controls, Menus,
  StdCtrls, cxButtons, cxCheckBox, cxContainer, cxEdit, cxTextEdit,
  cxMaskEdit, cxSpinEdit, ExtCtrls, cxGridLevel, cxGridCustomTableView,
  cxGridCardView, cxGridDBCardView, cxClasses, cxControls,
  cxGridCustomView, cxGrid, ComCtrls, cxStyles, cxCustomData, cxGraphics,
  cxFilter, cxData, DB, cxDBData, cxDataStorage, cxLookAndFeelPainters,
  cxLookAndFeels, cxHyperLinkEdit, cxImageComboBox, cxDBLookupComboBox,
  cxMemo, cxGridTableView, cxGridBandedTableView, cxBlobEdit, cxImage,
  cxCurrencyEdit, BaseForm, cxNavigator;

type
  TUnboundDesignDefinedDemoMainForm = class(TfmBaseForm)
    lvCars: TcxGridLevel;
    cxGrid: TcxGrid;
    bvCars: TcxGridBandedTableView;
    clnCar: TcxGridBandedColumn;
    clnPrice: TcxGridBandedColumn;
    clnPicture: TcxGridBandedColumn;
    clnHP: TcxGridBandedColumn;
    clnTorque: TcxGridBandedColumn;
    clnCyl: TcxGridBandedColumn;
    clnTransmissSpeedCount: TcxGridBandedColumn;
    clnTransmissAutomatic: TcxGridBandedColumn;
    clnHyperlink: TcxGridBandedColumn;
    cxStyleRepository1: TcxStyleRepository;
    styCar: TcxStyle;
    clnTrademark: TcxGridBandedColumn;
    styGroup: TcxStyle;
  end;

var
  UnboundDesignDefinedDemoMainForm: TUnboundDesignDefinedDemoMainForm;

implementation

{$R *.dfm}

end.

File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/CBuilder/UnboundDesignDefinedDemo/UnboundDesignDefinedDemoMain.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#include "shellapi.hpp"
#pragma hdrstop

#include "UnboundDesignDefinedDemoMain.h"
#include "AboutDemoForm.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "cxClasses"
#pragma link "cxControls"
#pragma link "cxCustomData"
#pragma link "cxData"
#pragma link "cxDBData"
#pragma link "cxEdit"
#pragma link "cxFilter"
#pragma link "cxGraphics"
#pragma link "cxGrid"
#pragma link "cxGridCustomTableView"
#pragma link "cxGridCustomView"
#pragma link "cxGridDBTableView"
#pragma link "cxGridLevel"
#pragma link "cxGridTableView"
#pragma link "cxStyles"
#pragma link "cxButtons"
#pragma link "cxCheckBox"
#pragma link "cxContainer"
#pragma link "cxGridCardView"
#pragma link "cxGridDBCardView"
#pragma link "cxLookAndFeelPainters"
#pragma link "cxMaskEdit"
#pragma link "cxSpinEdit"
#pragma link "cxTextEdit"
#pragma link "cxLookAndFeels"
#pragma link "cxGridBandedTableView"
#pragma link "cxCurrencyEdit"
#pragma link "cxDataStorage"
#pragma link "cxHyperLinkEdit"
#pragma link "cxImage"
#pragma link "cxImageComboBox"
#pragma resource "*.dfm"
TUnboundDesignDefinedDemoMainForm *UnboundDesignDefinedDemoMainForm;
//---------------------------------------------------------------------------
__fastcall TUnboundDesignDefinedDemoMainForm::TUnboundDesignDefinedDemoMainForm(TComponent* Owner)
  : TfmBaseForm(Owner)
{
}

